backend/question-service/server/infra/Pulumi.staging.yaml

```diff
@@ -1,15 +1,22 @@
 config:
   aws:region: ap-southeast-1
-  question-service-infra:MONGO_HOST:
-    secure: AAABALkzV+US7nKD68RP1FUXONRK8ryHT6lz36hwVDe624UFXmMa+BCqa9V7mJhD1e4JNF/gmXGtgnGgmxn9fdhA+eewmlm3FPf/gpYBVpmrgELReaga1Xh4dbCo
-  question-service-infra:MONGO_PORT:
-    secure: AAABAPFwD+A1Nfi4ez17NVKrV1dLTSnvlSeB2pZAE84ekTmI
+  question-service-infra:MONGO_ATLAS_DB_NAME: "questions_db"
+  question-service-infra:MONGO_ATLAS_PASSWORD:
+    secure: AAABALIssvhPWrYaJaczgurLtzCMWs3iAa2eB7zPkyC/xR+X1FpO5Q==
+  question-service-infra:MONGO_ATLAS_USERNAME:
+    secure: AAABAA3rhpwWQzE5+UvCInNP832nUVtoxHzzV4yz+x6v90JYMw==
+  question-service-infra:MONGODB_ATLAS_ORG_ID:
+    secure: AAABAPfcFwF432gMO6EvcOXUjKIGPKZ70P5TQmmzrTUzNkV5ZMqQT+sa/I0D46Ieykx8JLnaWxQ=
+  question-service-infra:MONGODB_ATLAS_PRIVATE_KEY:
+    secure: AAABAND3sHpA1YIdi9eFcXOwkWBhcJ9Lo58ZpO/Vptzsz8+4dIHPEN9RgewgpxOiZoBaBSF4eTUdvs1MAYUfvXrU6vI=
+  question-service-infra:MONGODB_ATLAS_PUBLIC_KEY:
+    secure: AAABAGwKo/qbWQhfy0JMpBMTd4ls95k4hLkG1ZAsOIEilSub1Fwe3Q==
   question-service-infra:acmEcsCertificateArn:
     secure: AAABAL3u5v31yQUyf5uyt3JXgX/p/WPHDHc1bGg5h4Uts4wSKDYFYj21z/H4Chqam/ru9zzAizQTEzX6D6/N2HSj3UTOtKkCr0sU50OLOvDYOs0QLq7ddVuIJBbbzp7Mh6upprFOseeoWrFHK5926wa41rxTImGY
   question-service-infra:containerPort: "80"
   question-service-infra:cpu: "512"
   question-service-infra:hostedZoneId:
     secure: AAABAMJRo8I7u45mjDQoh/ZCIKYHCszeT4OAKLxLK5QjOtmSx6hyGlXlaAbSfTJAMJjGBA==
   question-service-infra:memory: "128"
-  question-service-infra:rootDomain: peerprep.net
+  question-service-infra:domainName: peerprep.net
```

Comment:
btw is there a reason why you changed it from `rootDomain` to `domainName`? cos all the other services are using the convention of `rootDomain`

Reply:
In `index.ts` in template-service and every other service:

```
// For custom domain
const subdomain = config.get("subdomain") || "template.staging."; // TODO: Update `template` placeholder
const rootDomain = config.get("domainName") || "peerprep.net";
```

I changed it to domainName to work here, but I think this is an error with the `index.ts` line instead of the config so I will update mine accordingly - But template-service and every other service needs to change